decred_wallet/Features/Security/RequestPinViewController.swift

```diff
@@ -76,15 +77,26 @@ class RequestPinViewController: SecurityBaseViewController {
             self.pinInputView.clear()
             self.headerText.text = "Confirm \(self.securityFor) PIN"
             self.prgsPinStrength.progress = 0
+            
+            // We are confirming pin, we hide the pin strength meter
```

Comment:
`// We are confirming pin, hide the pin strength meter`

Reply:
Fixed in commit d097f900b6d316d6272559d520696d0fec66710a